tests/integration/main.js

```diff
@@ -1960,6 +2021,33 @@ test("subscribe to same room twice in quick succession, only one hook fired", t
   t.timeoutAfter(TEST_TIMEOUT)
 })
 
+test("message_deleted hook should fire if a message has been deleted", t => {
+  fetchUser(t, "alice")
+    .then(alice =>
+      alice
+        .sendSimpleMessage({ roomId: carolsRoom.id, text: "yo2" })
+        .then(sentMessageId => {
+          alice
+            .subscribeToRoom({
+              roomId: carolsRoom.id,
+              onMessageDeleted: deletedMessageId => {
+                t.equal(sentMessageId, deletedMessageId)
+              },
+            })
+            .then(room => {
+              deleteMessageV5(room.id, sentMessageId)
+                .then()
+                .catch(endWithErr())
+              alice.disconnect()
+              t.end()
```

Comment:
```
alice.disconnect()
t.end()
```
these want to be in the `onMessageDeleted` callback, otherwise the test will end before the hook even fires.